_docs/time-automation-explained.md

```diff
@@ -14,9 +14,9 @@ The APIs and Polkadot{.js} libraries on this page allow users and multi-chain ap
  - paying over time (recurring payments on a foreign chain),
  - anything else you can do on supported parachains, sometime(s) in the future.
 
-1. `scheduleDynamicDispatchTask` can be used to schedule execution of any extrinsic or batch of extrinsics on-chain.
+2. `scheduleDynamicDispatchTask` can be used to schedule execution of any extrinsic or batch of extrinsics on-chain.
 
-1. `cancelTask` can be used to cancel any existing automation tasks (must be from the account which created the task). 
+3. `cancelTask` can be used to cancel any existing automation tasks (must be from the account which created the task). 
```

Comment:
I think the three "1." work, that in display they will be shown as 1. 2. 3.

But there are pros and cons of suing using that markdown syntax, and I’m okay with explicitly label the items.